contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -21,13 +21,24 @@ pragma solidity 0.4.18;
 import "zeppelin-solidity/contracts/math/SafeMath.sol";
 import "../DebtRegistry.sol";
 import "../TermsContract.sol";
+import "ethereum-datetime/contracts/DateTime.sol";
```

Comment:
For now, since we're not including any `DateTime` specific functionality, let's not include this import.

Reply:
Agreed